multimodal/src/autogluon/multimodal/configs/optimization/adamw.yaml

```diff
@@ -17,6 +17,7 @@ optimization:
   gradient_clip_algorithm: "norm"
   track_grad_norm: -1 # Whether to check gradient norm. We can set it to 2 to check for gradient norm.
   log_every_n_steps: 10
+  val_metric: null
```

Comment:
I suggest to use `validation_metric` to make it more distinguishable from the already existing `eval_metric`.

Reply:
Oh sure, I'll change this in next pr